### ISSUE TITLE
关于msdbn.py文件的疑问

### ISSUE THREAD

**Issue by 895318** [Xiaosigz/MSDBN#1]
请问msdbn.py文件第21行import models.configs as configs中configs文件是哪一个呢？好像找不到。
并且该文件720行的resnet1.resnet50是不是应该是resnet.resnet50？
最后一个问题就是按照代码逻辑该文件745行的if labels is not None:的条件判断是否不执行？如果需要执行的话，貌似缺失了许多变量，比如logits、all_tokens等等。

谢谢！
